libs/remix-ui/debugger-ui/src/lib/vm-debugger/assembly-items.tsx

```diff
@@ -30,15 +38,108 @@ export const AssemblyItems = ({ registerEvent }) => {
       if (currentItem.firstChild) {
         currentItem.firstChild.removeAttribute('style')
       }
-      const codeView = asmItemsRef.current
+    }
+
+    currentItem = refs.current[nextSelectedItem] ? refs.current[nextSelectedItem] : null
+
+    if (currentItem) {
```

Comment:
This can refactored to be less repeatitive.
```
    if (currentItem) {
      currentItem.removeAttribute('selected')
      currentItem.removeAttribute('style')
      if (currentItem.firstChild) {
        currentItem.firstChild.removeAttribute('style')
      }
    }
```